src/test/java/emissary/test/core/junit5/RegressionTestUtil.java

```diff
@@ -95,6 +95,13 @@ public final class RegressionTestUtil {
      */
     private static final Path TEST_RESX = getTestResx();
 
+    /**
+     * Used for RegressionTest generateAnswer when {@link UnitTest#getMyTestParameterFiles(Class, Class)} has two different
+     * classes, and we need to generate answer files in the ansClass directory
+     */
+    @Nullable
+    private static String ansClass = null;
```

Comment:
This isn't thread-safe.  We can't use a static member to store state that multiple test classes need to manage independently.

Reply:
This and the `List<String> answerFiles` in UnitTest, correct?

Reply:
Yep - I'll have a `UnitTest` PR up shortly.  Hopefully it'll also make RegresstionTest changes easy